stdlib/runtime/numbers.gr

```diff
@@ -1415,6 +1534,8 @@ export let numberMod = (%);
 // inc/dec
 
 export let incr = (x) => {
+  Memory.incRef(WasmI32.fromGrain((+)))
```

Comment:
Why does `(+)` never get decref in here?

Reply:
See other replies